pkg/extender/extender.go

```diff
@@ -1053,21 +1019,11 @@ func (e *Extender) updateVirtLauncherPodPrioritizeScores(
 							// Current node has the volume replica
 							score = replicaNodeScore
 							nodeHasAttachedVolume := e.nodeHasAttachedVolume(dnode, volInfo)
-
-							if podBeingLiveMigrated != nil {
-								// LiveMigration in progress
-								// If existing pod is not using attached volume and current node has attached volume, assign a higher score to it
-								if !isExistingPodUsingLocallyAttachedVolume && nodeHasAttachedVolume {
-									storklog.PodLog(pod).Infof("Node %v has attached volume", dataNode)
-									score = int64(2 * nodePriorityScore)
-								}
-							} else {
-								if nodeHasAttachedVolume {
-									// There is no LiveMigration in progress
-									// When a new pod gets scheduling request
-									// assign a higher score for local volume attachment
-									score = int64(2 * nodePriorityScore)
-								}
+							if nodeHasAttachedVolume {
```

Comment:
nit: no need for the variable anymore. We can do `if e.nodeHasAttachedVolume(dnode, volInfo)` since it is called only once.

Reply:
Ack.